versioned_docs/version-v2.18.x/user-guide/configure-zowe-zosmf-workflow.md

```diff
@@ -74,35 +77,35 @@ Perform the following steps to execute each sub-step individually:
    ![Enabling components](../images/zosmf/workflow-componentsVariables.png)
 
    :::note
-   Ensure that all the components that you enabled are configured in the next sub-steps of **Define variables** step.
-   The sub-steps for configuring components will be in the Ready state, and the remainder of components in the Skipped state.
+   Ensure that all the components that you enabled are configured in the next sub-steps of the **Define variables** step.
+   The sub-steps for configuring components will be in the **Ready** state, and the remainder of components in the **Skipped** state.
    :::
 
 4. Select **Next**.
 5. Repeat the previous two steps to complete all items until the **Finish** option is available.
 
 :::note
-A basic validation is supported in many of the fields like a proper path structure, dataset name conventions, or numeric size.
-But the workflow does not check, for example,  whether a target dataset exists, or a directory has sufficient space.
+Basic validation is supported in many of the fields including the proper path structure, dataset name conventions, or numeric size.
+The workflow, however, does not check, for example, if a target dataset exists, or if a directory has sufficient space.
 :::
 
-After all sub-steps are completed the step **Define variables** is marked as Complete.
+After all sub-steps are completed, the step **Define variables** is marked as Complete.
 
 ### 2. **Create configuration**
 
-Execute this step to create a configuration zowe.yaml file with the variable setup that was defined in the previous step.
-In this step you can review your configurations and if needed you can make further changes directly in the JCL.
-When you are done, click Finish and the zowe.yaml file will be ready, and the step will be marked as Complete.
+This step creates a configuration zowe.yaml file with the variable setup that was defined in step 1.
+In this step you can review your configurations and, if necessary, you can make further changes directly in the JCL.
```

Comment:
I think we need a screenshot and steps describing where and how to perform this step.